README.md

```diff
@@ -310,13 +310,16 @@ proportion, including:
 
 | LightGBM Parameter                                                                                 | CV Search Range | Parameter Description                                                              |
 |:---------------------------------------------------------------------------------------------------|:----------------|:-----------------------------------------------------------------------------------|
-| [num_leaves](https://lightgbm.readthedocs.io/en/latest/Parameters.html#num_leaves)                 | 50 - 2000       | Maximum number of leaves in each tree. Main parameter to control model complexity. |
+| [num_iterations](https://lightgbm.readthedocs.io/en/latest/Parameters.html#num_iterations)         | 100 - 2500      | NA                                                                                 |
+| [learning_rate](https://lightgbm.readthedocs.io/en/latest/Parameters.html#learning_rate)           | -3 - -0.4       | NA                                                                                 |
+| [max_bin](https://lightgbm.readthedocs.io/en/latest/Parameters.html#max_bin)                       | 50 - 512        | Maximum number of bins used to bucket continuous features                          |
+| [num_leaves](https://lightgbm.readthedocs.io/en/latest/Parameters.html#num_leaves)                 | 32 - 2048       | Maximum number of leaves in each tree. Main parameter to control model complexity. |
```

Comment:
I think these params got added in https://github.com/ccao-data/model-res-avm/pull/172 but we didn't regenerate the README, so the diff is showing up now.